supervisor/addons/validate.py

```diff
@@ -285,6 +286,7 @@ def _migrate(config: dict[str, Any]):
         vol.Optional(ATTR_HOST_NETWORK, default=False): vol.Boolean(),
         vol.Optional(ATTR_HOST_PID, default=False): vol.Boolean(),
         vol.Optional(ATTR_HOST_IPC, default=False): vol.Boolean(),
+        vol.Optional(ATTR_HOST_UTS, default=False): vol.Boolean(),
```

Comment:
All other host namespace access results in a -1 of the add-on score, should that be done here?

Reply:
Maybe.

Host [UTS namespace](https://man7.org/linux/man-pages/man7/uts_namespaces.7.html) doesn't give you much, other than the hostname and hostdomain.

Security wise, it only really gets interesting once you also have `CAP_SYS_ADMIN`. `CAP_SYS_ADMIN` is also known as "the new root", as it actually gives you all the power (see [capabilities](https://man7.org/linux/man-pages/man7/capabilities.7.html) for a list of things). It seems that currently `CAP_SYS_ADMIN` gives -1, I'd argue that it should be more penalizing than that, before penalize UTS.

We could also make it conditional: if `CAP_SYS_ADMIN` is set, then UTS gives -1, thoughts?

Reply:
That said, I am biased, I am protective of OTBR's score :sweat_smile:

Reply:
Im not sure, was just mentioning that we usually do -1 
`CAP_SYS_ADMIN` might deserve -2 🤷‍♂️

Reply:
PID namespace `host` allows killing any process immediately without additional privileges (it seems that `CAP_KILL` is given by default). 

I implemented my suggested option.